### PR TITLE
Add installation command for requests to circleci

### PR DESCRIPTION
**Pull request by vilyapilya**
request is not installed in cci.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ jobs:
           name: Publish
           command: |
             sudo pip install twine
+            sudo pip install requests
```
Comment (smusali) [inline .circleci/config.yml]: why don't we have all of these in `requirements.txt`?

**Comment by dchai76** [comment]
A little confused here - the PR title says you're adding `requests`, but that's gone now. Now we have both a `pip` and `pip3` command, which is a little odd to me - we're building on both Python2 and Python3 simultaneously?

File: .circleci/config.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ jobs:
           name: Publish
           command: |
             sudo pip install twine
+            sudo pip3 install twine
```
Comment (smusali) [inline .circleci/config.yml]: as @dchai76 said, why are you having both of `pip install` and `pip3 install` in the same job?

File: .circleci/config.yml [hunk 3]
```diff
@@ -10,17 +10,20 @@ jobs:
...
+              echo "  PKG_VERSION_TOML: ${PKG_VERSION_TOML}"
               exit 1
             fi
       - run:
           name: Publish
           command: |
-            sudo pip install twine
-            python setup.py sdist
+            curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+            poetry build
```
Comment (vilyapilya) [inline .circleci/config.yml]: There is a command  `poetry publish --build`. I tested this command with test.pipy and it did not upload anything. So I am using poetry for creating the dist files and twine for uploading.

**Pull request closed without merging**